components/BarcodeScanner/index.js

```diff
@@ -18,68 +17,36 @@ const StyledCanvas = styled.canvas`
   height: 100%;
 `;
 
-const StyledImage = styled(Image)`
-  width: 25%;
-  height: 25%;
-  object-fit: cover;
-`;
 export default function BarcodeScanner() {
-  const [scanning, setScanning] = useState(true);
+  const [, setScanning] = useState(true);
```

Comment:
does this work? i thought you need a const name for the setter? 🧐

Reply:
yes it works because scanning is not used i only left the comma , when you remove the comma the setScanning is not a function